openshift/4.0/templates/maintenance/README.md

```diff
@@ -0,0 +1,88 @@
+# Maintenance Web Server Caddy
+
+The proxy caddy provides a way to redirec traffic temporarily during a maintenance period.
```

Comment:
This doc is not technically correct and needs to be updated once I get an opportunity to read and understand Devin's documentation on the Caddy.